packages/GoogleAnalytics/dist/types/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import { InitializeOptions } from 'react-ga';
+import { RouteTracker } from './components/GARoute';
+import WithGATracker from './components/WithGATracker';
+import { Dimensions, initGoogleAnalytics, setDimensions } from './helpers';
+export {
+  WithGATracker,
+  RouteTracker,
+  initGoogleAnalytics,
+  Dimensions,
+  InitializeOptions,
+  setDimensions
+};
```

Comment:
Thoughts on this:

- Consider exporting using [barrels](https://basarat.gitbook.io/typescript/main-1/barrel)
- This will auto-export everything that is exported from the linked files e.g. `'./components/GARoute'`
- For example, currently you have some things exported in `'./components/GARoute'` but not here.  Which means they are harder to discover and import for users of this package.  Is this intentional?
- Finally, I would consider not using default exports in the linked modules such as `'./components/GARoute'` so that you make working with barrel export easier.  [Default exports are also just generally controversial in Typescript](https://basarat.gitbook.io/typescript/main-1/defaultisbad).

cc: @cKellyDesign @kahummer @p-netm @ciremusyoka to share thoughts if they have on the above

Reply:
@moshthepitt Nice article

Reply:
@moshthepitt Agreed, let's make this a frontend standard! (issue: coming soon)

Reply:
Yes!

Reply:
https://github.com/onaio/ona-design-spec/issues/16